src/helpers/gutter/gutter.js

```diff
@@ -9,11 +9,11 @@ import { themeProvider } from '../../theme';
 const { theme } = themeProvider;
 
 export const gutterWidth = (props, breakpoint) => {
+  if (has(props, `gutter.${breakpoint}`)) {
```

Comment:
Yeah something _like_ this but Travis disagrees. Probably need to check if it's an object first or something.